set.mm

```diff
@@ -20290,11 +20291,12 @@ requires either a dv condition ( ~ sb56 ) or a non-freeness hypothesis
   ${
     $d x z $.
     $( An equation between setvar is free of any other setvar.  (Contributed by
-       Wolf Lammen, 9-Jun-2019.)  (Proof shortened by Wolf Lammen,
-       9-Jul-2022.) $)
+       Wolf Lammen, 9-Jun-2019.)  Remove dependency on ~ ax-12 .  (Revised by
+       Wolf Lammen, 20-Jul-2022.) $)
     nfeqf2 $p |- ( -. A. x x = y -> F/ x z = y ) $=
-      ( weq wal wn wex wnf exnal nfnf1 ax13lem2 ax13lem1 syld nfd exlimi sylbir
-      ) ABDZAEFQFZAGCBDZAHZQAIRTASAJRSARSAGSSAEABCKABCLMNOP $.
+      ( weq wal wn wex wnf exnal hbe1 ax13lem2 ax13lem1 syldc com12 hbe1a syl56
+      aleximi nfd sylbir ) ABDZAEFTFZAGZCBDZAHTAIUBUCAUCAGZUDAEZUBUCAEZAGZUFUCA
+      JUEUBUGUDUAUFAUAUDUCUFABCKABCLMQNUCAOPRS $.
```

Comment:
That's nice !
I just noticed that your hbe1a is identical to http://us2.metamath.org/mpeuni/bj-modal5e.html. No need to modify the credits. Proofs look the same size ? I'll uniformize the comments.